### PR TITLE
[resource] Use LinkedHashMap instead of LinkedList

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

File: biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceImpl.java [hunk 1]
```diff
@@ -51,7 +51,8 @@ void setCapabilities(List<Capability> capabilities) {
 
 	@Override
 	public List<Capability> getCapabilities(String namespace) {
```
Comment (pkriens) [inline biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceImpl.java]: We seem to basically reimplement MultiMap? Shouldn't we use that one and do any optimizations there?

Comment (bjhargrave) [reply biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceImpl.java]: Maybe a separate PR? But best not to conflate with this PR.

File: biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceBuilder.java [hunk 2]
```diff
@@ -124,17 +123,17 @@ public Resource build() {
 			throw new IllegalStateException("Resource already built");
 		built = true;
 
-		resource.setCapabilities(capabilities);
-		resource.setRequirements(requirements);
+		resource.setCapabilities(getCapabilities());
```
[the remark this answers is not in the document]
Comment (bjhargrave) [reply biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceBuilder.java]: Yes, I would just change the package private set methods in ResourceImpl to take the value `capabilities.values()`/`requirements.values()`.

**Comment by pkriens** [comment]
LGTM

**Pull request merged**